build/x86_64_centos6/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos:6
+
+ENV PATH ${PATH}:/usr/local/bin
```

Comment:
Just curious: why is this not needed on Debian? Is this where `rvm` lives on CentOS?

Reply:
I think it crept in. The build works fine. removed it for now.